src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2731,7 +2731,7 @@ internal override StrongNameKeys StrongNameKeys
                 if (moduleBeingBuilt.SourceModule.HasBadAttributes)
                 {
                     // If there were errors but no declaration diagnostics, explicitly add a "Failed to emit module" error.
-                    diagnostics.Add(ErrorCode.ERR_ModuleEmitFailure, NoLocation.Singleton, ((Cci.INamedEntity)moduleBeingBuilt).Name);
+                    diagnostics.Add(ErrorCode.ERR_ModuleEmitFailure, NoLocation.Singleton, ((Cci.INamedEntity)moduleBeingBuilt).Name, CodeAnalysisResources.ModuleHasInvalidAttributes);
```

Comment:
>CodeAnalysisResources.ModuleHasInvalidAttributes [](start = 134, length = 48)

This is likely to cause localization problems, I think we are using LocalizableResourceString in situations like this.